scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -0,0 +1,241 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    haproxy:
+      public_name: admin
+    stonith:
+      mode: sbd
+      sbd:
+        nodes:
+          "@@controller1@@":
+            devices:
+            - "/dev/sda"
+          "@@controller2@@":
+            devices:
+            - "/dev/sda"
+          "@@controller3@@":
+            devices:
+            - "/dev/sda"
+      per_node:
+        nodes:
+          "@@controller1@@":
+            params: ''
+          "@@controller2@@":
+            params: ''
+          "@@controller3@@":
+            params: ''
+  deployment:
+    elements:
+      pacemaker-cluster-member:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      hawk-server:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+
```

Comment:
Visual separation of the barclamp sections, having it all together makes it hard for visual parsing.